"No puzzle" issue solved, README updated (#1)
near-examples/near-crossword: README.md

   
   After this call your puzzle will be added to the NEAR Crossword contract. Share your Crossword with friends, the person who will be able to solve it will be able to generate the same key pair and get the reward. Let's do that in the following steps.

> 6. Pretend that we have solved the puzzle and generated the very same key pair. This time it should be stored at `~/.near-credentials/testnet/<contract-id>.json`. We are using `<contract-id>` here because in the next step we will need to sign transaction with this acc.

Reviewer: ```suggestion
6. Pretend that we have solved the puzzle and generated the very same key pair. This time it should be stored at `~/.near-credentials/testnet/<contract-id>.json`. We are using `<contract-id>` here because in the next step we will need to sign the transaction with this acc.
```

> Atension! If you are using the same machine, your old key pair from `<dev-acc>` will be owerwriten! Save it in some other place if you need it. Keys are stored in `~/.near-credentials/testnet/` folder.

Reviewer 2: This part is a bad `devx`. I was forced to move the key pair back and forth during the development. We really need a way to manage multiple keys.

Reviewer 3: 1. :nit: :typo: `Attention`

2. Do we really _delete the existing key without prompting the user_?  This seems _really_ dangerous, if not outright negligent, to the point that I would consider this a (critical) bug, if so.   IMO, we should _never_ delete a secret key, _ever_, without telling the user _and forcing them to confirm they really want this to happen via an e.g. --force argument_.

Reviewer 2: The problem is that we can not store more than one key on the machine. Now we are trying to come up with a better design.
I agree about the prompt, I have created an issue about it a few days ago: https://github.com/near/near-cli/issues/786

Reviewer: ```suggestion
Attention! If you are using the same machine, your old key pair from `<dev-acc>` will be overwritten! Save it in some other place if you need it. Keys are stored in `~/.near-credentials/testnet/` folder.
```

Puzzle solved! Let's get our reward!

> 8. To get the reward we need to call `claim_reward` function with the function call key that we have added in the prevoius step. Before that call we should prepare the keys:

Reviewer: ```suggestion
8. To get the reward we need to call the `claim_reward` function with the function call key that we have added in the previous step. Before that call we should prepare the keys:
```
